tests/Paramore.Brighter.RMQ.Async.Tests/MessagingGateway/Proactor/When_creating_quorum_queue_validation.cs

```diff
@@ -0,0 +1,122 @@
+    public void When_creating_quorum_consumer_without_durability_should_throw()
+    {
+        var rmqConnection = new RmqMessagingGatewayConnection
+        {
+            AmpqUri = new AmqpUriSpecification(new Uri("amqp://guest:guest@localhost:5672/%2f")),
+            Exchange = new Exchange("paramore.brighter.exchange")
+        };
+
+        var queueName = new ChannelName(Guid.NewGuid().ToString());
+        var routingKey = new RoutingKey(Guid.NewGuid().ToString());
+
+        var exception = Assert.Throws<ConfigurationException>(() =>
+            new RmqMessageConsumer(rmqConnection, queueName, routingKey,
+                isDurable: false, // This should cause the exception
+                highAvailability: false,
+                queueType: QueueType.Quorum));
+
+        Assert.Contains("Quorum queues require durability to be enabled", exception.Message);
+    }
+
+    [Fact]
+    public void When_creating_quorum_consumer_with_high_availability_should_throw()
+    {
+        var rmqConnection = new RmqMessagingGatewayConnection
+        {
+            AmpqUri = new AmqpUriSpecification(new Uri("amqp://guest:guest@localhost:5672/%2f")),
+            Exchange = new Exchange("paramore.brighter.exchange")
+        };
+
+        var queueName = new ChannelName(Guid.NewGuid().ToString());
+        var routingKey = new RoutingKey(Guid.NewGuid().ToString());
+
+        var exception = Assert.Throws<ConfigurationException>(() =>
+            new RmqMessageConsumer(rmqConnection, queueName, routingKey,
+                isDurable: true,
+                highAvailability: true, // This should cause the exception
+                queueType: QueueType.Quorum));
+
+        Assert.Contains("Quorum queues do not support high availability mirroring", exception.Message);
+    }
+
+    [Fact]
+    public void When_creating_quorum_consumer_with_correct_settings_should_succeed()
+    {
+        var rmqConnection = new RmqMessagingGatewayConnection
+        {
+            AmpqUri = new AmqpUriSpecification(new Uri("amqp://guest:guest@localhost:5672/%2f")),
+            Exchange = new Exchange("paramore.brighter.exchange")
+        };
+
+        var queueName = new ChannelName(Guid.NewGuid().ToString());
+        var routingKey = new RoutingKey(Guid.NewGuid().ToString());
+
+        // This should not throw any exception
+        using var consumer = new RmqMessageConsumer(rmqConnection, queueName, routingKey,
+            isDurable: true, // Required for quorum
+            highAvailability: false, // Must be false for quorum
+            queueType: QueueType.Quorum);
+
+        Assert.NotNull(consumer);
+    }
+
+    [Fact]
+    public void When_creating_classic_consumer_with_default_settings_should_succeed()
+    {
+        var rmqConnection = new RmqMessagingGatewayConnection
+        {
+            AmpqUri = new AmqpUriSpecification(new Uri("amqp://guest:guest@localhost:5672/%2f")),
+            Exchange = new Exchange("paramore.brighter.exchange")
```

Comment:
We need to make the Exchange Durable.

`
Exchange = new Exchange("paramore.brighter.durableexchange", durable: true)
`